h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -936,7 +936,12 @@ public Chunk chunkForChunkIdx(int cidx) {
    *  constructing Strings.
    *  @return {@code i}th element as {@link BufferedString} or null if missing, or
    *  throw if not a String */
-  public final BufferedString atStr( BufferedString bStr, long i ) { return chunkForRow(i).atStr_abs(bStr, i); }
+  public final BufferedString atStr( BufferedString bStr, long i ) {
+    if (isCategorical()) { //for categorical vecs, return the factor level
```

Comment:
This is a change in the core - I think it makes sense to ask a categorical Vec for its string value.
Reason I need this: For small files, a string column can be parsed as an enum due to the parse heuristic, and I don't want to transform the entire Vec to a String just because of that.